### DISCUSSION TITLE
What types of records are allowed?

### DISCUSSION THREAD

**Discussion by DEV-DIBSTER** [is-a-dev/register#1898] [Q&A] [answered]
So recently in [this pull request](https://github.com/is-a-dev/register/pull/1887) I made, there seemed to be confusion over the types of record one subdomain/pull request could have. My understanding is that we can have either `A`, `CNAME`, or `URL` as unique, and additionally have `MX` and one `TXT`. Someone correct me if I'm wrong.

**Comment by MaskDuck** [comment, marked as answer]
uh
CNAME: ok but no other record allowed
URL: ok
A: ok, just put it as array
MX: ok if you dont have any cname (if you have multiple MXs put it as an array)
TXT: ok if you dont have any cname (only 1 allowed)

**Reply by DEV-DIBSTER** [reply]
Alright thank you! Was questioning how that worked.

**Reply by MaskDuck** [reply]
np

**Reply by MaskDuck** [reply]
uh sorry i was wrong, i have edited the message through.

**Reply by MaskDuck** [reply]
@DIBSTERYT

**Reply by DEV-DIBSTER** [reply]
Alright just checked, thanks for the clarification.
